libs/structdiff/structpath/path.go

```diff
@@ -18,9 +19,10 @@ const (
 // PathNode represents a node in a path for struct diffing.
 // It can represent struct fields, map keys, or array/slice indices.
 type PathNode struct {
-	prev    *PathNode
-	jsonTag structtag.JSONTag // For lazy JSON key resolution
-	key     string            // Computed key (JSON key for structs, string key for maps, or Go field name for fallback)
+	prev      *PathNode
+	jsonTag   structtag.JSONTag // For lazy JSON key resolution
+	bundleTag structtag.BundleTag
```

Comment:
Both tags seem to be computed/resolved eagerly, not lazily. Is this comment incorrect?